frontend/src/scenes/notebooks/Notebook/Editor.tsx

```diff
@@ -64,14 +66,15 @@ export function Editor({
                 },
             }),
             NotebookNodeLink,
-
+            NotebookNodeBacklink,
```

Comment:
We already have NodeLink which is _almost_ identical. could we maybe merge them together? I believe it would be somewhat straightforward. 

Generally I prefer the style of your component so would maybe use that as a base and then pull the Link stuff in so that it supports any pasted links as well from around PH?

Reply:
Generally agree here. The only reason I didn't do that was because we won't have a title for the pasted link in the same way that we have a name for the referenced resource when inserted.

I was considering converting the existing links to be marks instead of nodes but wanted to check to see if there was a reason you'd done it the opposite way before making the change? Typically text manipulations (e.g. bold, italics, links) are all marks. That way we could insert the pasted link but people would be able to edit the text rendered themselves. The thing that would be missing is the icon that currently gets prepended to pasted nodes

Reply:
No strong reason for anything I did before tbh. I think I just went for the quickest approach 😅 
The only thought I remember having was if we include titles then we have a caching issue where the title may change and the link won't get updated...

So whichever approach you think is best I would vote for. If we add pasteability then we can remove the other link and only have your one. For pure links, we could simply show the minimal path and then perhaps later add a way for it to refresh the title via the API or something.